canonical-web-and-design/ubuntu.com
# add /internet-of-things/networking

## Done

- added /networking page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/internet-of-things/networking
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/6253#issuecomment-565471173) and the [copy doc](https://docs.google.com/document/d/1EWlkTkGvul-ek4P1cR9pyNvyJsh5WtpOkChvk0igzVQ/edit#)
- See also that "Networking" is available and highlighted in the secondary navigation, and also in the Enterprise dropdown navigation (per [copy doc](https://docs.google.com/document/d/1YBdQvLuqEpEQr_QqyycxMhZr4OaLapMlNJyYKCmPJsQ/edit))


## Issue / Card

Fixes #6254 & #6261
